source/utilities.tex

```diff
@@ -8573,7 +8587,7 @@
 
 \pnum
 \constraints
-\tcode{is_constructible_v<T, decltype(**this)>} is \tcode{true}.
+\tcode{is_constructible_v<T, decltype(\exposid{val})>} is \tcode{true}.
```

Comment:
There's extra parentheses in the PR.
Although that shouldn't make a difference.
Feel free to reject, but do tell Thomas,
as this might be worth raising in the editor's report.
```suggestion
\tcode{is_constructible_v<T, decltype((\exposid{val}))>} is \tcode{true}.
```

Reply:
Leaving this one to @tkoeppe :). FYI - this is LWG3973.

Reply:
The extra parentheses are correct and removing them will break the spec, please don't.

`decltype(val)` here is the declared type of the member, i.e. `T`. If we wanted that we would just write `T`.

`decltype((val))` is affected by the cv- and ref-qualifiers of the member function, so it's either `T&` or `const T&` depending which overload it appears in. This is what we want.

Reply:
fixed, thanks!

Reply:
Right, I wasn't thinking about that.
I guess I've adopted the mind that the Paper is Final.